src/App.css

```diff
@@ -28,6 +28,34 @@
   color: #61dafb;
 }
 
+.title {
+  padding: 24px 24px 24px 24px;
+}
+
+.container {
+  display: flex;
+  height: 30px;
+  margin-bottom: 16px;
+  text-align: center;
+  padding: 0px 0px 0px 24px;
+}
+
+
+.id-container {
+  width:30px; 
```

Comment:
Both the `width` and the `height` should be `48px`

![image](https://user-images.githubusercontent.com/37349324/210094076-d794c362-9ef8-49cf-97b8-cf5d6d1a0280.png)